src/backend/decoder_adapter.cpp

```diff
@@ -65,10 +65,10 @@ void DecoderAdapter::addtoFrame(uint8_t *v)
         }
     }
 
-    decoder->Feed(data, length);
+    decoder->Feed(&data[0], length);
```

Comment:
And equivalent way is `data.data()`

Reply:
I vote also changing. It makes the code more readable.
@gvanem Please change it.

Reply:
I give up on this horrid Git-interface. Now Git won't allow me to edit any of these files:
![git-error](https://user-images.githubusercontent.com/945271/76162341-15829480-613d-11ea-8731-427be86362c6.png)

But if you merge this PR, I'll try to make a new PR on the `next` branch.

Reply:
If you do all your changes through the github web interface, I'm not surprised that you find it horrid to use.

Maybe https://desktop.github.com/ or https://www.sourcetreeapp.com/ could be useful to you, or the command-line interface https://www.atlassian.com/git/tutorials